src/components/common/multi-entry/multi-entry.styles.tsx

```diff
@@ -1,10 +1,12 @@
 import { Accordion } from "@lifesg/react-design-system/accordion";
 import { ButtonWithIcon } from "@lifesg/react-design-system/button-with-icon";
+import { Color } from "@lifesg/react-design-system/color";
 import styled from "styled-components";
 
 export const MultiEntryAccordionItem = styled(Accordion.Item)`
-    padding-bottom: 1rem;
     width: 100%;
+    border-top: 1px solid ${Color.Neutral[6]};
+    border-bottom: 1px solid ${Color.Neutral[6]};
```

Comment:
- top border is not needed
- remove top border from `AccordionWrapper` from `element-editor` too

 so we will rely on the top border that comes with `Accordion.Item` in `FieldEditorAccordionItem`

```suggestion
    border-top: 0;
    border-bottom: 1px solid ${Color.Neutral[6]};
```